### PR TITLE
Update tensorboard_profiler_tutorial.py

### PR DESCRIPTION
**Pull request by cirquit**
Added Safari incompatibility to torch-tb-profiler documentation.

Does not fix the issue https://github.com/pytorch/pytorch/issues/30525, but warns users about it as I wasted some time on debugging this problem.

## Description
Updated the documentation on the torch-tb-profiler to warn about the Safari incompatibility.

## Checklist

- [ ] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [x] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [x] No unnecessary issues are included into this pull request.

cc @aaronenyeshi @chaekit @jcarreiro @sekyondaMeta @svekars @carljparker @NicolasHug @kit1980 @subramen

### CODE REVIEW

**Review by aaronenyeshi — APPROVED** [review]
LGTM!

File: intermediate_source/tensorboard_profiler_tutorial.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ def train(data):
 #
 
 ######################################################################
-# Open the TensorBoard profile URL in Google Chrome browser or Microsoft Edge browser.
+# Open the TensorBoard profile URL in Google Chrome browser or Microsoft Edge browser (**Safari is not supported** [#30525](https://github.com/pytorch/pytorch/issues/30525))
```
Comment (svekars) [inline intermediate_source/tensorboard_profiler_tutorial.py:177]:
```suggestion
# Open the TensorBoard profile URL in Google Chrome browser or Microsoft Edge browser (**Safari is not supported**)
```

**Review by svekars — APPROVED** [review]

File: intermediate_source/tensorboard_profiler_tutorial.py [hunk 2]
```diff
@@ -178,7 +178,7 @@ def train(data):
 #
 
 ######################################################################
-# Open the TensorBoard profile URL in Google Chrome browser or Microsoft Edge browser.
+# Open the TensorBoard profile URL in Google Chrome browser or Microsoft Edge browser (**Safari is not supported**)
```
Comment (svekars) [inline intermediate_source/tensorboard_profiler_tutorial.py:181]:
```suggestion
# Open the TensorBoard profile URL in Google Chrome browser or Microsoft Edge browser (**Safari is not supported**).
```
